test/extensions_testsuite/dpl_namespace.pass.cpp

```diff
@@ -40,13 +39,12 @@ int main()
     auto res_first = dpl::begin(res_buf);
     auto counting_first = dpl::counting_iterator<T>(0);
     auto zip_first = dpl::make_zip_iterator(counting_first, key_first);
- 
+
     // key_buf = {0,0,...0,1,1,...,1}
     std::for_each(dpl::execution::make_device_policy<class ForEach>(dpl::execution::dpcpp_default),
 		zip_first, zip_first + n,
-        [](auto x){
-            using std::get;
-            get<1>(x) = (2 * get<0>(x)) / n;
+        [](std::tuple<T, T> x){
+            std::get<1>(x) = (2 * std::get<0>(x)) / n;
```

Comment:
was there the goal to use ADL call? I mean it would work even if you don't rewrite it to `std::get`. I am just trying to make sure that we don't introduce new problems

Reply:
Both cases should work ("red" and "green"). Changes just demonstrate that we don't need to write write `using std::get` anymore.

Reply:
I think both variants are fine. I just wanted to make sure, that we don't decrease some previously tested use-cases. On the other hand, `T` is `uint64_t` so I think that changes you did are fine.